monai/utils/type_conversion.py

```diff
@@ -301,3 +301,15 @@ def convert_to_dst_type(
     else:
         output_type = type(dst)
     return convert_data_type(data=src, output_type=output_type, device=device, dtype=dtype, wrap_sequence=wrap_sequence)
+
+
+def convert_to_list(data: Union[Sequence, torch.Tensor, np.ndarray]) -> list:
```

Comment:
is a list really needed here? perhaps this could be done with the existing `ensure_tuple` utility?
```py
>>> from monai.utils import ensure_tuple
>>> ensure_tuple(torch.zeros((3, 4)))
(tensor([0., 0., 0., 0.]), tensor([0., 0., 0., 0.]), tensor([0., 0., 0., 0.]))
>>> import numpy as np
>>> ensure_tuple(np.zeros((3, 4)))                                                                                                                                
(array([0., 0., 0., 0.]), array([0., 0., 0., 0.]), array([0., 0., 0., 0.]))
```

Reply:
> is a list really needed here? perhaps this could be done with the existing `ensure_tuple` utility?
> 
> ```python
> >>> from monai.utils import ensure_tuple
> >>> ensure_tuple(torch.zeros((3, 4)))
> (tensor([0., 0., 0., 0.]), tensor([0., 0., 0., 0.]), tensor([0., 0., 0., 0.]))
> >>> import numpy as np
> >>> ensure_tuple(np.zeros((3, 4)))                                                                                                                                
> (array([0., 0., 0., 0.]), array([0., 0., 0., 0.]), array([0., 0., 0., 0.]))
> ```
I tried and got
```
>>> ensure_tuple(torch.zeros((3, )))
(tensor(0.), tensor(0.), tensor(0.))
```
I would expect 
```
>>> convert_to_list(torch.zeros((3, )))
[0., 0., 0.]
```
May be it's OK to not use it. But I need to test the code to know

Reply:
sure, I'll merge this PR for now but please check, intuitively the module should also support `(tensor(0.), tensor(0.), tensor(0.))`, it'll be useful for the refactoring for metatensor